src/main/java/moze_intel/projecte/utils/GuiHandler.java

```diff
@@ -129,7 +128,7 @@ public Object getServerGuiElement(int ID, EntityPlayer player, World world, int
 			case Constants.MERCURIAL_GUI:
 				return new MercurialEyeContainer(player.inventory, new MercurialEyeInventory(player.getHeldItem(hand)));
 			case Constants.PHILOS_STONE_GUI:
-				return new PhilosStoneContainer(player.inventory);
+				break;
```

Comment:
just remove this constant and the `case`

Reply:
I didn’t originally, but I opted for keeping them to preserve the number order and to make it easier to add something new in the future. Removing them is simple enough though.